commit 979ffc416cadfcf6f9518e342f03d973e76e968b
Author: Sergei Trofimovich <slyfox@gentoo.org>
Date:   Tue Feb 11 22:16:38 2014 +0300

    rts/package.conf.in: fix UNREG on --with-system-libffi when include-dir is passed explicitely
    
    Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>

diff --git a/rts/package.conf.in b/rts/package.conf.in
index 010305f..eb38690 100644
--- a/rts/package.conf.in
+++ b/rts/package.conf.in
@@ -57,7 +57,7 @@ extra-libraries:
 #endif
 
 #ifdef INSTALLING
-include-dirs:           INCLUDE_DIR PAPI_INCLUDE_DIR
+include-dirs:           INCLUDE_DIR PAPI_INCLUDE_DIR FFI_INCLUDE_DIR
 #else /* !INSTALLING */
-include-dirs:           TOP"/rts/dist/build" TOP"/includes" TOP"/includes/dist-derivedconstants/header"
+include-dirs:           TOP"/rts/dist/build" TOP"/includes" TOP"/includes/dist-derivedconstants/header" FFI_INCLUDE_DIR
 #endif
